examples/plugins/src/lighthouse/src/lighthouse.plugin.unit.test.ts

```diff
@@ -0,0 +1,78 @@
+import { describe, expect, it } from 'vitest';
+import { LIGHTHOUSE_URL } from '../mock/constants';
+import { LIGHTHOUSE_OUTPUT_FILE_DEFAULT } from './constants';
+import { runnerConfig } from './lighthouse.plugin';
+import type { LighthouseCliOptions } from './types';
+
+describe('lighthouse-runnerConfig', () => {
+  const baseOptions: LighthouseCliOptions = {
+    url: LIGHTHOUSE_URL,
+  };
+  const lcpAuditOutputBase = {
+    displayValue: expect.stringContaining('sec'),
+    score: 1,
+    slug: 'largest-contentful-paint',
+    value: 0,
+  };
+
+  it('should execute if url is given', () => {
+    expect(runnerConfig(baseOptions)).toEqual(
+      expect.objectContaining({
+        args: expect.arrayContaining(['lighthouse', LIGHTHOUSE_URL]),
+        command: 'npx',
+        outputFile: LIGHTHOUSE_OUTPUT_FILE_DEFAULT,
+      }),
+    );
+  });
+
+  it('should execute if with output options', () => {
```

Comment:
I don't understand the purpose of this test case. The only property in `baseOptions` variable is the URL, no output options are given as far as I can see.